core/eolearn/tests/test_utils/test_parsing.py

```diff
@@ -181,6 +166,15 @@ def test_allowed_feature_types_iterable(test_input: FeaturesSpecification, allow
         FeatureParser(features=test_input, allowed_feature_types=allowed_types)
 
 
+@pytest.fixture(name="eopatch", scope="module")
+def eopatch_fixture():
+    patch = generate_eopatch(
+        {FeatureType.DATA: ["data", "CLP"], FeatureType.MASK: ["data", "IS_VALID"], FeatureType.MASK_TIMELESS: ["LULC"]}
+    )
+    patch.meta_info = {"something": "else"}
```

Comment:
i see this comes up a few times :thinking: 
If you want you can play with the `generate_eopatch` to also create dummy  `meta_info`. But for that I'd keep it very simple, like `patch[random_meta_feature] = _random_string(rng)` and
```python
import string

def _random_string(rng):
    random_letters = rng.choice(string.ascii_lettes, 20)
    return "".join(random_letters)
```

If you think that is not worth it, we can keep it as it is